### ISSUE TITLE
ads.exefolderpath returns full folder path or module path ?

### ISSUE THREAD

**Issue by JohnWins** [Ahmadmansoor/AdvancedScript#9]
hi, new question.

> ads.exefolderpath			            get exe folder path

From help file, thought that return value should not contain executable name, but everytime it does not. Such as: 

> varx str,vFolder
setx $vFolder,ads.exefolderpath
setx $vFolder,ads.exename
==> C:\Windows\notepad.exe

**Comment by Ahmadmansoor** [comment]
hmmm
I think I agree with u in this.
I will update it today.
if u have any others thing's(bugs)pls let me know before I upload the update.
BR

**Comment by JohnWins** [comment]
yep, found another one.
>setx $vFolder,ads.exefolderpath
> real path: C:\\{app}\x64\target.exe
>return path: C:\\???x64\target.exe

"{app}" string is not visible .
maybe the script system consider it as a variable or something different.

**Comment by Ahmadmansoor** [comment]
yes it is (the script system considers it as a value need to analyzed)
I don't know how to pass it now, but this will check it later.
